### DISCUSSION TITLE
Javers working with ReactiveMongoRepository?

### DISCUSSION THREAD

**Discussion by dalvan-bevilaqua** [javers/javers#1197] [Q&A]
I Have an method to save in my Service
`
@JaversAuditableAsync
  public void save(Object object) {
    repository.save(object).subscribe();
  }`

and my Repository
`public interface MyRepository extends ReactiveMongoRepository<Object , ObjectId> {}`

I not find in documentation the configuration to async method, Javers work with is configuration?

**Comment by bartoszwalacik** [comment]
well, this is still incubating feature and I have never tested it with reactive Spring. could you please debug it? see JaversAuditableAspectAsync
